src/main/scala/edu/ie3/mobsim/model/ElectricVehicle.scala

```diff
@@ -401,18 +396,16 @@ case object ElectricVehicle extends LazyLogging {
       simulationStart = simulationStart,
       uuid = UUID.randomUUID(),
       id = id,
-      model = s"${evType.producer} ${evType.model}",
-      batteryCapacity = evType.capacity,
-      acChargingPower = evType.acPower,
-      dcChargingPower =
-        if (
-          evType.dcPower.isLessThan(
-            Quantities.getQuantity(0.1, PowerSystemUnits.KILOWATT)
-          )
-        )
-          evType.acPower
-        else evType.dcPower,
-      consumption = evType.consumption,
+      // todo: check if this is neccessary
```

Comment:
This does look kind of odd, but since it was in there before it's fine for me to leave it in for now